SPEC/STATS.md

```diff
@@ -25,7 +25,11 @@ Where:
   - `poll` is used to print bandwidth at an interval.
   - `interval` is the time interval to wait between updating output, if `poll` is true.
 
-`callback` must follow `function (err, stats) {}` signature, where `err` is an error if the operation was not successful. `stats` is an Object containing the following keys:
+If `poll` is `true`, then `callback` must follow `function (err, stream) {}` signature, where `err` is an error if the operation was not successful and `stream` is a Readable Stream where you can listen to the event `data` with a listener that must follow `function (stat) {}` signature.
```

Comment:
Why ReadableStream and not pull-stream?

Reply:
I can use a pull-stream, but I didn't find a way [to do this](https://github.com/ipfs/js-ipfs-api/pull/686/files#diff-7b55c8c4d8fa3f0c172a6eaa8f2a80fdR39) with that package...

Reply:
Also, `js-ipfs-api` doesn't seem to be using pull streams. at least the package isn't a dependency.

Reply:
And how can we close a pull-stream? Their docs say nothing about it. The Readable Stream seems to be more straightforward and easy to use. You just have to do `.on('data', fn)`.